imap_processing/cdf/config/imap_enamaps_l2-common_variable_attrs.yaml

```
@@ -196,6 +196,50 @@ exposure_factor: &exposure_factor
   DISPLAY_TYPE: no_plot
   DICT_KEY: SPASE>TemporalDescription:Exposure,Qualifier:Directional,CoordinateSystemName:HAE,CoordinateRepresentation:Spherical
 
+geometric_function:
```

I think this should be geometric factor?
```suggestion
geometric_factor:
```

---

I had the same thought but apparently they want geometric function. Will that be going against the grain? Maybe it is worth it to have a separate ultra l2 map yaml file.